### PR TITLE
Update PatchAlerts note for both v1.0 & beta versions

### PR DESCRIPTION
**Pull request by chinguyen1**

### CODE REVIEW

File: api-reference/beta/resources/security-api-overview.md [hunk 1]
```diff
@@ -26,21 +26,24 @@ The Microsoft Graph Security API includes the following key entities.
...
+|[Microsoft Defender Advanced Threat Protection](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/attack-simulations) ***| <p align="center">&#x2713;</p> | <p align="center">&#x2713;</p> | <p align="center"> [File issue](https://github.com/microsoftgraph/security-api-solutions/issues/new) </p> |
+|[Azure Advanced Threat Protection](https://docs.microsoft.com/azure-advanced-threat-protection/understanding-security-alerts#security-alert-categories) **| <p align="center">&#x2713;</p> | <p align="center">[File issue](https://github.com/microsoftgraph/security-api-solutions/issues/new) *</p> | <p align="center">&#x2713;</p> |
 |Office 365 <ul><li> [Default](https://docs.microsoft.com/office365/securitycompliance/alert-policies#default-alert-policies)</li> <li>[Cloud App Security](https://docs.microsoft.com/office365/securitycompliance/anomaly-detection-policies-in-ocas)</li></ul> | <p align="center">&#x2713;</p> | <p align="center"> [File issue](https://github.com/microsoftgraph/security-api-solutions/issues/new) </p> | <p align="center"> [File issue](https://github.com/microsoftgraph/security-api-solutions/issues/new) </p> |
```
Comment (preetikr) [inline api-reference/beta/resources/security-api-overview.md:40]: Add (Coming soon) under file issue for this one (only PATCH)

**Comment by Lauragra** [comment]
@preetikr , can you approve this PR, or does it still need changes? Thanks.

**Comment by Lauragra** [comment]
@chinguyen1 , @preetikr , this is set to ready to merge but Preeti has requested changes. Do you want me to merge, or are you still making updates? Thanks.

**Comment by chinguyen1** [comment]
I'm opening a new PR to include cleaner changes.

**Pull request closed without merging**